osconfig_tests/test_suites/guestpolicies/guest_policies.go

```diff
@@ -90,6 +98,43 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	testSuite := junitxml.NewTestSuite(testSuiteName)
 	defer testSuite.Finish(testSuites)
 
+	storage, err := gcpclients.GetStorageClient()
+	if err != nil {
+		logger.Printf("error retrieving storage client: %s", err)
+		logger.Printf("skipping test suite %s due to error", testSuiteName)
+		return
+	}
+	bucket := storage.Bucket(tempBucketName)
```

Comment:
Can we use a common bucket for all tests, just a temp 'folder', since buckets are in a shared namespace I don't want this to go crazy. You can leave the create call if it doesnt exist, just don't make it unique and there is no need to clean it up after.

Reply:
The issue with using the same bucket name every time is that bucket names are global so if different people try to run the tests the names will collide.

Reply:
True, can you generate it based on the project the tests run in? that way its just one bucket per project

Reply:
we can use one bucket so long as the subdir is unique and has collision detection. but generating a bucket based on the project name is pretty standard. should do both imo, so two running in one project won't collide, and different projects will use different buckets

Reply:
i think this is the only outstanding change blocking this PR. for most of the other software we use or maintain that dumps to GCS, it typically assumes creating a bucket in the project you're running in for the application overall (so e.g. osconfig-tests or similar), with file naming based on the test run. e.g. gs://${project}-osconfig-tests/testrun-${date}/ or similar. bucket can remain existing after test completion, but you can remove files within it.